### ISSUE TITLE
drizzle-kit@0.19.9 Regression

### ISSUE THREAD

**Issue by bestickley** [drizzle-team/drizzle-kit-mirror#131]
Starting in drizzle-kit@0.19.9 I've found a regression in drizzle-kit not being able to find tables correctly. The logs below explain in detail, but with drizzle-kit@0.19.9 when I run `generate:pg` I get a migration that wants to delete all my tables. But with drizzle-kit@0.19.8 I get the correct migration.

I'm using a PNPM Monorepo and I've ensured I only have one installation of drizzle-kit.

drizzle.config.ts
```ts
import type { Config } from "drizzle-kit";

export default {
  out: `./src/db/drizzle`,
  schema: `./src/modules/**/*.db.ts`,
} satisfies Config;

```

Command Line Logs:
```bash
pnpm add drizzle-kit@0.19.9
../infra                                 |  WARN  deprecated @aws-cdk/core@1.204.0
../infra                                 |  WARN  deprecated querystring@0.2.0
../ui                                    |  WARN  deprecated sourcemap-codec@1.4.8
../infra                                 |  WARN  deprecated @aws-cdk/cloud-assembly-schema@1.204.0
../infra                                 |  WARN  deprecated @aws-cdk/cx-api@1.204.0
../infra                                 |  WARN  deprecated @aws-cdk/region-info@1.204.0
..                                       |   +1   -1 +-
..                                       | Progress: resolved 1037, reused 1008, downloaded 0, added 0, done
.. prepare$ husky install
│ husky - Git hooks installed
└─ Done in 339ms

dependencies:
- drizzle-kit 0.19.8
+ drizzle-kit 0.19.9 (0.19.10 is available)

Done in 9.7s
pnpm tsx node_modules/drizzle-kit/index.cjs generate:pg --config ./src/db/drizzle.config.ts
drizzle-kit: v0.19.9
drizzle-orm: v0.27.2

Reading config file '/Users/stickb/Code/bayer/seed-sight/core/src/db/drizzle.config.ts'
0 tables


[✓] Your SQL migration file ➜ src/db/drizzle/0013_friendly_wallflower.sql 🚀
pnpm tsx node_modules/drizzle-kit/index.cjs drop --config ./src/db/drizzle.config.ts
drizzle-kit: v0.19.9
drizzle-orm: v0.27.2

Reading config file '/Users/stickb/Code/bayer/seed-sight/core/src/db/drizzle.config.ts'

[✓] 0013_friendly_wallflower migration successfully dropped
pnpm add drizzle-kit@0.19.8
../infra                                 |  WARN  deprecated @aws-cdk/core@1.204.0
../infra                                 |  WARN  deprecated querystring@0.2.0
../ui                                    |  WARN  deprecated sourcemap-codec@1.4.8
../infra                                 |  WARN  deprecated @aws-cdk/cloud-assembly-schema@1.204.0
../infra                                 |  WARN  deprecated @aws-cdk/region-info@1.204.0
../infra                                 |  WARN  deprecated @aws-cdk/cx-api@1.204.0
..                                       |   +1   -1 +-
..                                       | Progress: resolved 1037, reused 1008, downloaded 0, added 0, done
.. prepare$ husky install
│ husky - Git hooks installed
└─ Done in 422ms

dependencies:
- drizzle-kit 0.19.9
+ drizzle-kit 0.19.8 (0.19.10 is available)

Done in 9.9s
pnpm tsx node_modules/drizzle-kit/index.cjs generate:pg --config ./src/db/drizzle.config.ts
drizzle-kit: v0.19.8
drizzle-orm: v0.27.2

Reading config file '/Users/stickb/Code/bayer/seed-sight/core/src/db/drizzle.config.ts'
3 tables
batch 31 columns 0 indexes 2 fks
batches_upload 8 columns 0 indexes 2 fks
user_account 7 columns 0 indexes 0 fks

[✓] Your SQL migration file ➜ src/db/drizzle/0013_puzzling_nova.sql 🚀
```

**Comment by AndriiSherman** [comment]
Please try `drizzle-kit@fixes-pg` tag. Should fix this issue and I'll push it to the latest today

**Comment by AndriiSherman** [comment]
Fixed in `drizzle-kit@0.19.11`
